File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapMethods.java
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat;

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapMethod;

import java.lang.annotation.*;

@Documented
@Target(ElementType.FIELD)
@Retention(RetentionPolicy.RUNTIME)
public @interface WrapMethods {
    WrapMethod[] value();
}

File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapMethods.kt
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapMethod

@MustBeDocumented
@Target(AnnotationTarget.FIELD)
@Retention(AnnotationRetention.RUNTIME)
annotation class WrapMethods(vararg val value: WrapMethod)